requirements.txt

```diff
@@ -0,0 +1,11 @@
+astropy
+certifi                       # via requests
+chardet                       # via requests
+h5py
```

Comment:
I see setup.py requires >= 2.9

Reply:
That's fine, katsdpdockerbase provides 2.10 as the default pin.